tests/test_interfaces.py

```diff
@@ -778,6 +723,103 @@ def test_get_interface_mac_address(self):
 
         self.assertEqual(self.network_manager.get_interface_mac(interface_name), self.mac_address)
 
+    def test_get_interface_1_ap_1_mon_internet_interfaces(self):
```

Comment:
for user specifying their own cards like the following command:
`wifiphisher -aI wlan0 -jI wlan1 -iI wlan10`

we'll call the `is_interface_valid` three times in the `pywifiphisher.py`. I'm just thinking to simulate this condition. sorry if I have misused the name or doing wrong in the test.